src/link-handler.js

```diff
@@ -82,10 +82,10 @@ export class DefaultLinkHandler extends LinkHandler {
     info.anchor = target;
     info.href = href;
 
-    let hasModifierKey = (event.altKey || event.ctrlKey || event.metaKey || event.shiftKey);
+    let leftButtonClicked = event.which ? event.which === 1 : event.button & 1;
```

Comment:
Ah yes, I didn't see that line. Sounds good!
On Sun, Nov 29, 2015 at 12:10 PM Sergey Aksyonov <notifications@github.com>
wrote:

> In src/link-handler.js
> <https://github.com/aurelia/history-browser/pull/20#discussion_r46097229>:
>
> > @@ -82,10 +82,10 @@ export class DefaultLinkHandler extends LinkHandler {
> >      info.anchor = target;
> >      info.href = href;
> >
> > -    let hasModifierKey = (event.altKey || event.ctrlKey || event.metaKey || event.shiftKey);
> > +    let leftButtonClicked = event.which ? event.which === 1 : event.button & 1;
>
> Got it, we can check here only for event.which which is supported by IE9+
> and other browsers and will be equal to 1 for the main mouse button.
>
> About checks for meta keys - there is the same check on this line
> <https://github.com/aurelia/history-browser/blob/1a67ae11885db085d6921f78e0ca69afb616c883/src/link-handler.js#L77>
> and after that hasModifierKey will always be false. What do you think
> about it?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/aurelia/history-browser/pull/20/files#r46097229>.
>